s3torchconnectorclient/pyproject.toml

```diff
@@ -36,3 +37,50 @@ find = { where = ["python/src"] }
 
 [[tool.setuptools-rust.ext-modules]]
 target = "s3torchconnectorclient._mountpoint_s3_client"
+
+[tool.setuptools]
+license-files = [ "LICENSE", "THIRD-PARTY-LICENSES", "NOTICE"]
+
+[tool.cibuildwheel]
+test-requires = ["./s3torchconnector[e2e]"]
+test-extras = "test"
+test-command = [
+    "pytest {package}/python/tst/",
+    "pytest {package}/../s3torchconnector/tst",
+    "CI_STORAGE_CLASS=EXPRESS_ONEZONE CI_REGION=${CI_EXPRESS_REGION} CI_BUCKET=${CI_EXPRESS_BUCKET}  pytest {package}/python/tst/integration",
```

Comment:
These are running against S3 Express One Zone, where are we running against standard S3?
We also need CI_PREFIX to be set.

Reply:
The first two lines are running against S3 Standard.  CI_REGION/CI_BUCKET and their express counter part would be setup before executing that command. All of them will be exposed as part of `environment-pass` bellow in the file. Because we changed variable names for S3 Express integration tests to be the same as for S3 Standard, we need to override these variables, before starting tests for S3 Express, otherwise these variable will contains settings for S3 Standart.

Reply:
Sorry bit confused, when running integration-tests here, are we running them using artefacts of the build_wheels step or running them from source-code. 

In other words what are we testing: 1/We built wheels from a commit that is passing integration-tests. 2/We built a wheel that passes all the tests-cases specified in our integration tests?

Reply:
Sorry for confusion, we have a separate workflow to run only integration tests in `python-integration.yml`. It is run for pushing to main and for creating PR to main, so I think we are fine with separate workflow that has no dependency on building wheels.

Here we are building wheels and testing that these wheels are passing all the tests.

Reply:
> test-command = [
    "pytest {package}/python/tst/",
    "pytest {package}/../s3torchconnector/tst",
    "CI_STORAGE_CLASS=EXPRESS_ONEZONE CI_REGION=${CI_EXPRESS_REGION} CI_BUCKET=${CI_EXPRESS_BUCKET}  pytest {package}/python/tst/integration",
    "pytest {package}/../s3torchconnector/tst/e2e",
]
What I was trying to call out is that the way this is currently achieved is a bit confusing, although correct, as we are selecting just the subset of tests dependent on S3 Express setup. Do you think it would be worth to have a more verbose approach here?
i.e. unit tests command appears once, integration / e2e appears command appears once for S3, once for S3 Express.